### PR TITLE
Updated documentation with some examples on common errors

### PR DESCRIPTION
**Pull request by sauron**
Added some improvements in the documentation for the Installation an CRUD modules based on a step by step App creation.

### CODE REVIEW

**Comment by arkid** [comment]
Is it possible that the documentation as it ends up on the twill website includes a "Last updated" time stamp ?

**Comment by sauron** [comment]
@arkid Yes, that's possible.
Will add that timestamp on my next pull request.

Thanks for the suggestion.

File: docs/.sections/getting-started/installation.md [hunk 1]
```diff
@@ -58,6 +80,22 @@ Twill's `install` command consists of:
 - publishing Twill's assets for the admin console UI.
 - prompting you to create a superadmin user.
 
+##### Common Errors when running the install command:
+-Conneciton Error
```
Comment (droplister) [inline docs/.sections/getting-started/installation.md]: Nit: Conneciton -> Connection

**Comment by ifox** [comment]
Reviewed and merged as part of 09e22483554b574c0137c824829ac5fedc2cd33e. Thank you so much @sauron!

**Pull request closed without merging**